mautic/mautic
# [UX] Add new keyboard shortcuts and remap existing ones

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

![image](https://github.com/user-attachments/assets/00b6e875-f06f-44f1-bc10-ceadc69c5840)


This PR adds new keyboard shortcuts:
- Edit (when you're viewing a contact or anything editable)
- Send email (when you're viewing a Contact) 
- Send example (when you're viewing an Email)
- Save changes (when you're editing something)
- Back to list (when editing/viewing)
- Go to Segments
- Go to Landing pages
- Search current list
- Go to Forms page (great because they're contained in Components, so you reach there faster)
- Go to Emails page  (also nice bc they're contained in Channels)
- Create (on a list page, to create a resource)
- Delete (when you're viewing a resource)

It also remaps existing ones to improve the logic behind them.
- Navigation items (that bring you to another page) now use the prefix/key g (Go to). For example, `g` `d` to Go to Dashboard.
- Instead of having a key to activate card view and another key to activate table view for contacts, now we have only one V (view) key to toggle the view between card or table 








---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Tap Shift + / (or just "?") to open the shortcuts modal
3. Have fun testing each
